easybuild/easyconfigs/s/SRA-Toolkit/SRA-Toolkit-3.0.0-gompi-2021b.eb

```diff
@@ -0,0 +1,83 @@
+# updated: Denis Kristak (INUITS)
+easyblock = 'ConfigureMake'
+
+name = 'SRA-Toolkit'
+version = '3.0.0'
+
+homepage = 'https://github.com/ncbi/sra-tools'
+description = """The SRA Toolkit, and the source-code SRA System Development
+ Kit (SDK), will allow you to programmatically access data housed within SRA
+ and convert it from the SRA format"""
+github_account = 'ncbi'
+
+toolchain = {'name': 'gompi', 'version': '2021b'}
+toolchainopts = {'extra_cflags': '-lxml2 -lz -llzma'}
+
+source_urls = ['https://github.com/ncbi/sra-tools/archive/refs/tags/']
+sources = [{'download_filename': '%(version)s.tar.gz', 'filename': SOURCE_TAR_GZ}]
+patches = ['SRA-Toolkit-%(version)s_fix-LD_LIBRARY_PATH.patch']
+checksums = [
+    {'SRA-Toolkit-3.0.0.tar.gz': 'b6e8116ecb196b91d5ee404cc978a128eec9af24bdc96f57ff7ebfaf9059a760'},
+    {'SRA-Toolkit-3.0.0_fix-LD_LIBRARY_PATH.patch': '80b764584e377150cbc706463ef2c72e517d1a3e7493f13e26d6e7875cd7d16a'},
+]
+
+builddependencies = [
+    ('Perl', '5.34.0'),
+    ('Python', '3.9.6'),
+    ('CMake', '3.21.1'),
+    ('flex', '2.6.4'),
+    ('Bison', '3.7.6'),
+    ('Java', '11', '', SYSTEM),
+]
+
+dependencies = [
+    ('ncbi-vdb', '3.0.0'),
```

Comment:
@deniskristak Does `SRA-Toolkit` 3.0.0 effectively require a `ncbi-vdb` 3.0.0, or can we stick to `ncbi-vdb` 2.11.2 which is already used by various easyconfigs of the `2021b` generation?

Reply:
though I didn't find any specific place where it would be stated, I do tend to believe it is like that.
Firstly, it comes from the same developer and ships pretty much like one package.
Secondly, according to this line, it would probably be non-trivial as these two use different packaging systems:
![image](https://user-images.githubusercontent.com/35582739/210580920-a6306968-aeb2-452d-9c0c-b7df063a2fde.png)

Reply:
@deniskristak Let's go forward here by adding an exception in `test/easyconfigs/easyconfigs.py` to `ncbi-vdb` 3.0.0 being used as dependency for `SRA-Toolkit` 3.0.0 + `Finder`

Reply:
@boegel done, here: https://github.com/easybuilders/easybuild-easyconfigs/pull/17131